site/layout.html

```diff
@@ -127,11 +127,7 @@
             <label id="a11y-editor-label" class="v-visible-sr">
                 Stacks-Editor demo implementation
             </label>
-            {% block content %}
```

Comment:
I'll miss having this default example for reference, but... not so much that I'd block this change from going in.

lgtm